packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -423,7 +425,18 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({
               <div>
                 {(() => {
                   if (!getValues('primaryDiagnosisSearch')) return null;
-                  if (loadingPrimary)
+                  if (error) {
+                    return (
+                      <InlineNotification
+                        className={styles.errorNotification}
+                        lowContrast
+                        title={t('error', 'Error')}
+                        subtitle={t('errorFetchingConcepts', 'There was a problem fetching concepts') + '.'}
+                        onClose={() => setError(null)}
+                      />
+                    );
+                  }
+                  if (loadingPrimary) {
```

Comment:
Can we remove that (() => {})() and simply put if()... statements?

My concern here seems to be that we create a function, then we call it, which is redundant.

Reply:
Fixed in the latest commit.